exporters/otlp/otlptrace/otlptracegrpc/client_test.go

```diff
@@ -0,0 +1,839 @@
+// Copyright The OpenTelemetry Authors
```

Comment:
Copied from: otlpgrpc/otlp_integration_test.go.  Just bookkeeping.

Reply:
Sorry, I needed to keep track of this somewhere.  I didn't mean for you to add it.

Reply:
Hah, no problem :)